salt/cni/cilium-ds.yaml.jinja

```
@@ -158,9 +158,10 @@ spec:
             path: /etc/pki
       restartPolicy: Always
       tolerations:
-      - effect: NoSchedule
-        key: node-role.kubernetes.io/master
-      - effect: NoSchedule
-        key: node.cloudprovider.kubernetes.io/uninitialized
-        value: "true"
+        # Allow the pod to run on the master.  This is required for
+        # the master to communicate with pods.
+        - operator: Exists
```

Could we use a key more explicit like this ?

```suggestion
      tolerations:
      - key: node-role.kubernetes.io/master
        operator: Exists
        effect: NoSchedule

---

@lcavajani that looks much cleaner but didn't work on my setup. Also flannel is using the same, I guess it would make sense to do for both, test and create a separate PR. We need this PR for beta

---

I did try with a random DaemonSet and it works for me.

---

ok My bad I had some indentation issue before